filler.rb

```diff
@@ -0,0 +1,33 @@
+require 'ffaker'
+
+class Filler
+
+  @@new_lipsum = -> { FFaker::HTMLIpsum.body }
```

Comment:
Prefer either class instance variables or class constants to class (`@@`) variables ... they do some unexpected things when inheritance gets involved.